### PR TITLE
[STK-227] Pingdom support

### PR DESCRIPTION
**Pull request by waseem-h**

### CODE REVIEW

**Comment by stakater-user** [comment]
@waseem-h Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.tools.stackator.com/job/Stakater/job/IngressMonitorController/job/PR-55/1/) has Failed!

File: src/pingdom-monitor_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package main
+
+// import "testing"
```
Comment (hazim1093) [inline src/pingdom-monitor_test.go]: @waseem-h Why is this commented?

**Comment by stakater-user** [comment]
@waseem-h Image is available for testing. `docker pull stakater/ingressmonitorcontroller:PR-55-3`

File: src/pingdom-monitor_test.go (hunk 1 above)
Comment (waseem-h) [reply src/pingdom-monitor_test.go]: We cant run these tests in our pipeline. They require pingdom api key

Comment (hazim1093) [reply src/pingdom-monitor_test.go]: @rasheedamir are we good to go with this?

Comment (rasheedamir) [reply src/pingdom-monitor_test.go]: yeah we can't do anything!

File: docs/pingdom-configuration.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Pingdom Configuration
+
+Currently additional pingdom configurations can be added through a set of annotations to each ingress object, the current supported annotations are:
+
+|                        Annotation                        |                    Description                   |
+|:--------------------------------------------------------:|:------------------------------------------------:|
+| monitor.stakater.com/pingdom/resolution                  | The pingdom check interval in minutes            |
```
[the remark this answers is not in the document]
Comment (hazim1093) [reply docs/pingdom-configuration.md]: @drubin I've opened the issue here https://github.com/stakater/IngressMonitorController/issues/69.

**Pull request merged**